tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "noEmit": true,
```

Comment:
/cc @ricardogobbosouza I think here mistake

Reply:
no, that was intentional. tsc was attempting to overwrite source files without that and producing an error in vscode. There is an inverse of that on the npm script generating the type definitions.

Reply:
oh, yes, my mistake